app/javascript/controllers/form/hidden_inputs_controller.js

```diff
@@ -0,0 +1,26 @@
+import { Controller } from "@hotwired/stimulus";
+import { createHiddenInput } from "utilities/form";
+
+export default class extends Controller {
+    static outlets = ["selection"];
+    static values = {
+        fieldName: String,
+    };
+
+    connect() {
+        this.allIds = this.selectionOutlet.getStoredItems();
```

Comment:
**nitpick:** I know this has nothing to do with this PR, but can you update this?  There is no reason to keep all the stored items as an attribute on this controller, it is a waste of storage.  Maybe pass it as a variable or something into appendHiddenInputs.

Reply:
Fixed in [f64be2d](https://github.com/phac-nml/irida-next/pull/691/commits/f64be2d1ad8c8b297f1ae7e37a06c696c193484a)